### PR TITLE
Cleanup appium cs js sample

### PR DESCRIPTION
**Pull request by michal-przytarski**

### CODE REVIEW

File: samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md [hunk 1]
```diff
@@ -38,19 +42,38 @@ it contains information about project and what's more important - dependenencies
...
+    "appId": "YOUR_APPLICATION_ID"
+}
+```
 
 ### Prepare Webdriver.io Configuration
 
 You need to edit `wdio.conf.js` and set capabilities. You can either edit existing one (if you set API key then it should
-work out of the box) or you can use [Capabilities Creator](https://bitbar.github.io/capabilities-creator/) (you just need
-to switch mode to "Appium" and in top right corner language to "Node.js").
+work out of the box) or you can use [Capabilities Creator](https://staging.bitbar.com/#public/capabilities-creator) (you just need
```
Comment (srautiai) [inline samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md:71]: ~~staging~~ => cloud

File: samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md [hunk 2]
```diff
@@ -38,19 +42,38 @@ it contains information about project and what's more important - dependenencies
...
+```
 npm ci
 ```
+### Add your apiKey to `./.credentials.json`
+
+Create a file called ".credentials.json" in the project's root and add your testdroid apiKey to it as described below:
+
+```json
+{
+    "apiKey": "YOUR_TESTDROID_CLOUD_APIKEY"
```
Comment (srautiai) [inline samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md:54]: ~~TESTDROID~~ => BITBAR

File: samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md [hunk 3]
```diff
@@ -38,19 +42,38 @@ it contains information about project and what's more important - dependenencies
...
+}
+```
+
+### Add sample application ID to `./.credentials.json`
+Download [sample application](https://github.com/bitbar/test-samples/blob/master/apps/android/bitbar-sample-app.apk) and upload it to your BitBar Files Library. Copy applications ID and add it to  `.credentials.json` file:
+
+```json
+{
+    "apiKey": "YOUR_TESTDROID_CLOUD_APIKEY",
```
Comment (srautiai) [inline samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md:63]: ~~TESTDROID~~ => BITBAR

File: samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md [hunk 4]
```diff
@@ -7,23 +7,27 @@ Note: For now it's Android only example.
...
+└── .credentials.json
 ```
 
 * `package.json` and `package-lock.json` is obvious for everyone familiar with Node.js. Shortly for those who doesn't know:
 it contains information about project and what's more important - dependenencies.
+* `.npmrc` - Config for npm
 * `screenshots` - Here will appear all taken screenshots. Note: Directory will be cleared before each test run.
 * `test/specs/main.js` - Test itself
 * `wdio.conf.js` - Config for [Webdriver.io Testrunner](https://webdriver.io/docs/gettingstarted.html)
+* .`credentials.json` - This file isn't included in the repository, however it's required to run sample code. You need to create it by yourself according to instruictions below. It is used to store sensitive data.
```
Comment (srautiai) [inline samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md:30]: ~~instruictions~~

Comment (michal-przytarski) [reply samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md:30]: Thanks! done

File: samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md (hunk 2 above)
Comment (michal-przytarski) [reply samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md:54]: Thanks, done

File: samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md (hunk 3 above)
Comment (michal-przytarski) [reply samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md:63]: Thanks, done

File: samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md (hunk 1 above)
Comment (michal-przytarski) [reply samples/testing-frameworks/appium/client-side/javascript/webdriver.io/README.md:71]: Thanks, done

File: samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/nightwatch.json [hunk 5]
```diff
@@ -6,46 +6,45 @@
...
+                "automationName": "XCUITest",
                 "browserName": "safari",
-                "automationName": "XCUITest"
+                "bitbar_project": "Nightwatch_Example_IOS",
+                "bitbar_testrun": "TestRun_IOS"
             }
         },
-        "testdroid_android": {
+        "bitbar_android": {
             "launch_url": "https://appium.bitbar.com:80/wd/hub",
```
Comment (szprutamich) [inline samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/nightwatch.json:29]: the port should be removed

File: samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/.npmrc [hunk 6]
```diff
@@ -0,0 +1 @@
+engine-strict=true
```
Comment (szprutamich) [inline samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/.npmrc:1]: w new line would be great

Comment (michal-przytarski) [reply samples/testing-frameworks/appium/client-side/javascript/nightwatch_web_example/.npmrc:1]: OK, done. Thanks!

**Pull request merged**